code/modules/admin/verbs/warning.dm

```diff
@@ -334,6 +392,51 @@
 	usr << browse(dat, "window=lookupwarns;size=900x500")
 	feedback_add_details("admin_verb","WARN-LKUP")
 
+//Admin Proc to add a new User Notification
+/client/proc/notification_add()
+	set category = "Admin"
+	set name = "Add Notification"
+
+	if(!check_rights(R_ADMIN|R_MOD|R_DEBUG|R_DEV|R_CCIAA))
+		return
+
+	var/ckey = input(usr, "What ckey ?", "Enter a ckey")
+	if(!ckey)
+		to_chat(usr,"You need to specify a ckey.")
+		return
+
+	//Validate ckey
+	var/DBQuery/validatequery = dbcon.NewQuery("SELECT count(*) as count FROM ss13_player WHERE ckey = :ckey:")
+	validatequery.Execute(list("ckey" = ckey))
```

Comment:
Before this query, might want to do `ckey = ckey(ckey)`. It'll put the ckey into canonical form. Just a quality of life improvement, admins/mods/CCIAA don't necessarily know the difference between the key and ckey forms.